README.md

```
@@ -1,78 +1,421 @@
+   ```bash
+   pnpm dev
+   ```
+
+4. **ブラウザでアクセス**
+   - アプリケーション: http://localhost:3000
+   - API ドキュメント: http://localhost:3000/api/swagger
+   - OpenAPI 仕様: http://localhost:3000/api/openapi.json
+
+## 📁 プロジェクト構造
+
+```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)